File: CS/CompoundPrinting/Form1.cs
using System;
using DevExpress.XtraEditors;

namespace CompoundPrinting {
    public partial class Form1 :XtraForm {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            productsTableAdapter.Fill(nwindDataSet.Products);
            categoriesTableAdapter.Fill(nwindDataSet.Categories);
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            compositeLink.ShowPreview();
        }
    }
}

File: VB/CompoundPrinting/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors

Namespace CompoundPrinting
	Partial Public Class Form1
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			productsTableAdapter.Fill(nwindDataSet.Products)
			categoriesTableAdapter.Fill(nwindDataSet.Categories)
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			compositeLink.ShowPreview()
		End Sub
	End Class
End Namespace